packages/angular/src/generators/web-worker/lib/normalize-options.ts

```diff
@@ -1,10 +1,37 @@
 import type { WebWorkerGeneratorOptions } from '../schema';
+import type { Tree } from '@nrwl/devkit';
+import { getProjects, joinPathFragments } from '@nrwl/devkit';
 
 export function normalizeOptions(
+  tree: Tree,
   options: WebWorkerGeneratorOptions
 ): WebWorkerGeneratorOptions {
+  if (!options.project) {
+    throw new Error('You must provide a project!');
+  }
+
```

Comment:
This shouldn't be needed. The `project` option is required in the schema and it will be automatically validated. Same as the `name` which we are not validating here.
```suggestion
```